assets/js/frontend/single-product.js

```diff
@@ -108,6 +108,8 @@ jQuery( function( $ ) {
 		if ( this.flexslider_enabled ) {
 			this.initFlexslider();
 			$target.on( 'woocommerce_gallery_reset_slide_position', this.onResetSlidePosition );
+		} else {
+			$( '.woocommerce-product-gallery' ).css( 'opacity', 1 );
```

Comment:
@mikejolley -- Did you mean to use `$target` here? The object is agnostic to the classes that the gallery container has.